source/edge-server/configure.txt

```diff
@@ -490,40 +509,63 @@ To cycle the Edge Server API key:
 Specify the Edge Server's Query
 -------------------------------
 
-You can use the ``--query`` argument with the ``edgectl configure`` command
-to specify the Edge Server instance Sync query.
+If you don't specify a query, an Edge Server instance defaults to a global 
+query of ``"*"``, which automatically syncs all data in all of the collections
+in your linked data source.
+
+You can use a few arguments with the ``edgectl configure`` command
+to specify the Edge Server instance Sync query:
+
+- Pass a simple string query:
+
+  .. code-block:: console
+
+     edgectl config --query="Store: store_id == 42"
```

Comment:
i'm not certain but i think in RQL all of these are just one equals?

Reply:
Our [RQL docs](https://www.mongodb.com/docs/atlas/device-sdks/realm-query-language/#comparison-operators) show that both `==` and `=` are valid, and the examples all show `==`, so I've defaulted to the syntax that is consistent with the examples. If you think there's a reason to prefer the single `=`, though, I'm open to changing it.